Source/SelfService/Web/microservice/microserviceView/microserviceView.tsx

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { useReadable } from 'use-svelte-store';
+
+import { microservices, MicroserviceStore, canEditMicroservice } from '../../stores/microservice';
+import { Configuration } from './configuration';
+import { HttpResponsePodStatus } from '../../api/api';
+import { MicroserviceSimple } from '../../api/index';
+import { HttpResponseApplication } from '../../api/application';
+
+import { Box, Typography } from '@mui/material';
+
+import { Tabs } from '@dolittle/design-system/atoms/Tabs/Tabs';
+
+import { HealthStatus } from './healthStatus/healthStatus';
+import { ConstainerHealthStatus } from '../microserviceStatus';
+
+type MicroserviceViewProps = {
+    application: HttpResponseApplication;
+    environment: string;
+    microserviceId: string;
+    podsData: HttpResponsePodStatus;
+};
+
+export const MicroserviceView = ({ application, microserviceId, environment, podsData }: MicroserviceViewProps) => {
+    const $microservices = useReadable(microservices) as any;
+    const history = useHistory();
+
+    const getContainerStatuses = () => {
+        return podsData.pods.flatMap(pod => {
+            return pod.containers.map((container) => {
+                return container.state;
+            });
+        });
+    };
```

Comment:
If you want to simplify:
```suggestion
    const getContainerStatuses = () => podsData.pods.flatMap(pod => pod.containers.map(container => container.state));
    };
```